src/index.html

```diff
@@ -1,12 +1,390 @@
-<!doctype html>
+  <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="header">
+        <a
+          href="#home"
+          class="logo">
+          <img
+          src="images/logo.png"
+          alt="logo">
+        </a>
```

Comment:
```suggestion
        <a
          href="#home"
          class="logo"
        >
          <img
            src="images/logo.png"
            alt="logo"
          >
        </a>
```
Pay attention to formatting. Fix all cases